riverdriver/river_driver_interface.go

```diff
@@ -209,9 +209,19 @@ type Notification struct {
 }
 
 type JobCancelParams struct {
+	ID                int64
 	CancelAttemptedAt time.Time
 	ControlTopic      string
-	ID                int64
+}
+
+type JobCountByStateParams struct {
+	Schema string
```

Comment:
So I'd originally had `Schema` as a configuration parameter to the driver, but I realized later that I don't think that's appropriate ... not only is the UX a bit questionable, but it should really be only the client that controls its insert/work schema, and it should be possible for two clients to share a single driver and be using two separate schemas.

The slightly unfortunate part is that this approach would necessitate adding a `Schema string` parameter to every driver params structs where the underlying query targets a River table, but compared to the other possibilities, this might be the best/most explicit way to go.

Reply:
Yep, I think that's reasonable. Explicitly passing through the schema makes the most sense and also makes it clear the driver needs to handle running its queries on an arbitrary schema.